### ISSUE TITLE
Back end 

### ISSUE THREAD

**Issue by osamuca** [gabineteopensource/demandas-vitoria-mobile#2]
@stewwan você poderia contribuir com o backend do app?

**Comment by stewwan** [comment]
of course. mas se for firebase não tem muito oq fazer de backend, resolve tudo no app. se tiver algo mais "pesado" da pra rodar no functions. vc soh precisa definir uma especificação mais precisa, sugiro o uso do [sketch app](https://www.sketchapp.com/) pra esboçar a ideia

**Comment by osamuca** [comment]
@stewwan sketch app é pago né?

**Comment by stewwan** [comment]
ja tentou a doc do firebase?
https://firebase.google.com/docs/auth/web/facebook-login?hl=pt-br

tirar foto e escolher da lib, apenas este plugin ja resolve
https://ionicframework.com/docs/native/camera/
